src/DGtal/geometry/curves/estimation/LambdaMST3D.h

```diff
@@ -0,0 +1,186 @@
+/**
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ **/
+
+#pragma once
+
+/**
+ * @file LambdaMST3D.h
+ * @author Kacper Pluta (\c kacper.pluta@esiee.fr )
+ * Laboratoire d'Informatique Gaspard-Monge - LIGM, France
+ *
+ * @date 2014/10/06
+ *
+ * This file is part of the DGtal library.
+ */
+
+#if defined(LAMBDAMST3D_RECURSES)
+#error Recursive header files inclusion detected in LambdaMST3D.h
+#else // defined(LAMBDAMST3D_RECURSES)
+/** Prevents recursive inclusion of headers. */
+#define LAMBDAMST3D_RECURSES
+
+#if !defined LAMBDAMST3D_h
+/** Prevents repeated inclusion of headers. */
+#define LAMBDAMST3D_h
+
+#include <algorithm>
+#include <iterator>
+#include <cmath>
+#include <map>
+#include <DGtal/base/Common.h>
+#include <DGtal/helpers/StdDefs.h>
+#include "DGtal/kernel/CSpace.h"
+#include "DGtal/kernel/PointVector.h"
+#include "DGtal/geometry/curves/estimation/FunctorsLambdaMST.h"
+#include "DGtal/geometry/curves/CForwardSegmentComputer.h"
+#include "DGtal/geometry/curves/estimation/CLMSTTangentFromDSS.h"
+
+namespace DGtal {
+  /**
+   * Aim: Implement Lambda MST tangent estimator.
+   * @tparam TSpace model of CSpace
+   * @tparam TSegmentation tangential cover obtained by a segmentation of a 2D digital curve by maximal straight segments
+   * @tparam Functor model of CLMSTTangentFrom2DSS
+   */
```

Comment:
Would it be difficult to do ?
If not, I would prefer to have the estimator in the correct concept before merging.